examples/splunk-layer/template.yml

```diff
@@ -23,6 +23,6 @@ Resources:
         Variables:
           OTEL_INSTRUMENTATION_AWS_LAMBDA_HANDLER: com.splunk.support.lambda.examples.RequestFunction::handleRequest
           OTEL_INSTRUMENTATION_AWS_LAMBDA_FLUSH_TIMEOUT: 30
-          OTEL_EXPORTER: logging
+          OTEL_TRACE_EXPORTER: logging
```

Comment:
It's been changed to `OTEL_TRACES_EXPORTER`

Reply:
+1